test/testLocaleInfo.js

```diff
@@ -12820,8 +12820,8 @@ module.exports.testlocaleinfo = {
         var info = new LocaleInfo("ar-AE");
         test.ok(info !== null);
 
-        test.equal(info.getWeekEndStart(), 5);
-        test.equal(info.getWeekEndEnd(), 6);
+        test.equal(info.getWeekEndStart(), 6);
+        test.equal(info.getWeekEndEnd(), 0);
```

Comment:
When I checked cldr 41, For AE country, weeEndStart is Friday, and weekendEnd is Saturday. so modified test case is wrong.

Reply:
Where did you check?

Take a look: https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-core/supplemental/weekData.json

AE is not listed in the weekend start and end lists...

Also, a snippet from CLDR 41 XML in `common/supplemental/supplementalData.xml`:

```
               <firstDay day="sun" territories="GB" alt="variant" references="Shorter Oxford Dictionary (5th edition, 2002)"/>
                <weekendStart day="thu" territories="AF"/>
                <weekendStart day="fri" territories="BH DZ EG IL IQ IR JO KW LY OM QA SA SD SY YE"/>
                <weekendStart day="sat" territories="001"/>
                <weekendStart day="sun" territories="IN UG"/>
                <weekendEnd day="fri" territories="AF IR"/>
                <weekendEnd day="sat" territories="BH DZ EG IL IQ JO KW LY OM QA SA SD SY YE"/>
                <weekendEnd day="sun" territories="001"/>
```

That seems to have the same information as the json above.

Reply:
oh sorry. I checked the wrong file.. you are right.
Not all features of iLib were not updated to cldr 41, and due to that, there are test cases expectation is different. that's why I'm confused.